fourkitchens/suhumsci
# shs-4955: account for ck5 css bugs

## Summary
There are CSS bugs that ship with CK5, this fixes.

## Steps to Test
1. Go to a preexisting page with components. Click on 'Edit' for that component. 
2. First, check that the drag and drop arrow on the left hand side is no longer cut off. The cause for this was conflicting styles between CK5 and Drupal module styles.
3. Second, observe that the CK5 container is no longer overlapping into the right column. The cause for this is that CK5 has `nowrap` for its button layout. If there are too many buttons, like here, it will just keep going and going.

## Screenshots
![Screenshot 2023-06-08 at 9 06 51 AM](https://github.com/fourkitchens/suhumsci/assets/8405274/53b1ed78-39aa-4838-b170-b4477cd52a2f)
